package.json

```diff
@@ -51,7 +51,7 @@
     "core-js": "^3.6.5",
     "deprecated-prop-type": "^1.0.0",
     "downshift": "^5.4.7",
-    "formik": "^2.2.0",
+    "formik": "^2.4.6",
```

Comment:
Supports react 18 (with less issues)

Reply:
Should Formik be a peer dependency?

Reply:
no breaking changes in formik, all good?